File: course_2/code/oc/Student.m
//Student.m
#include"Student.h"

@implementation Student
-(void) study:(float)time
{
    printf("This student has studying for %f hours.\n", time);
    //@NSlog("This student has studying for %f hours.\n", time);
}
@end

int main(){
    NSLog(@"hello world");
    Student *s = [[Student alloc] init];
    [s study:3.0];
    return 0;
}

File: course_2/code/swift/Student.swift
//Student.swift
class Student {
    var name: String
    var major: String
    var age: Int
    init(name: String, major: String, age: Int) {
        self.name = name
        self.major = major
        self.age = age
    }
    study(time: float) {
        print("This student has studying for \(time) hours.")
    }
}